src/static/js/ace2_inner.js

```diff
@@ -2894,6 +2894,11 @@ function Ace2Inner(){
       rep.selFocusAtStart = newSelFocusAtStart;
       currentCallStack.repChanged = true;
 
+      hooks.callAll('aceSelectionChanged', {
+        rep: rep,
+        documentAttributeManager: documentAttributeManager,
```

Comment:
The plugin I intend to use this hook uses the attribute manager to look at the attributes of the current cursor to show the current style in a UI element.  Other hooks pass the documentAttributeManager in this way, so I thought that would be fine.